pyproject.toml

```diff
@@ -39,22 +39,28 @@ exclude = '''
 [tool.pylint.messages_control]
 disable=[
     "bad-option-value",
-    "line-too-long",
-    "invalid-name",
-    "too-few-public-methods",
-    "no-self-argument",
+    "broad-except",
     "dangerous-default-value",
     "duplicate-code",
-    "logging-fstring-interpolation",
+    "fixme",
     "import-error",
-    "unused-argument",
-    "no-self-use",
     "import-outside-toplevel",
-    "unsubscriptable-object", # Otherwise throws errors on certain Type annotations
+    "invalid-name",
+    "line-too-long",
+    "logging-fstring-interpolation",
+    "missing-class-docstring",
+    "missing-function-docstring",
     "missing-module-docstring",
+    "no-self-argument",
+    "no-self-use",
     "raise-missing-from",
+    "too-few-public-methods",
     "too-many-ancestors",
-    "fixme"]
+    "too-many-arguments",
+    "unnecessary-comprehension",
+    "unsubscriptable-object", # Otherwise throws errors on certain Type annotations
+    "unused-argument",
+]
```

Comment:
Any reason you've added these extra lint rules? We usually talk change management in our retrospectives.

Reply:
1. `broad-except`
	To eliminate linter errors thrown by code like:
	```python
	try:
		do_something()
	except Exception: # thrown here
		handle_any_exception()
	``` 
	Originally seen in several places throughout the `api` and `service` packages.
1. `missing-class-docstring`
	To eliminate linter errors thrown by...missing class docstrings. I don't have the context to author many of these.
1. `missing-function-docstring`
	Same as above.
1. `too-many-arguments`
	IMO, it's a silly guideline rule that happens to be enabled by default. It was throwing for a couple of functions.
1. `unnecessary-comprehension`
	The alternative would be to significantly change logic where this error was thrown, which I was not comfortable doing (at least not in this PR).